build-system/server/new-server/transforms/tsconfig.json

```diff
@@ -7,15 +7,19 @@
     "moduleResolution": "node",
     "target": "ES2019",
     "module": "CommonJS",
-    "allowJs": false,
+    "allowJs": true,
     "noUnusedLocals": true,
     "strictNullChecks": true,
     "noImplicitAny": true,
     "noImplicitThis": true,
     "alwaysStrict": true,
     "incremental": true,
     "esModuleInterop": true,
-    "declaration": true
+    "declaration": true,
+    "baseUrl": ".",
+    "paths": {
+      "../../../../tasks/": ["*", "../../../../../tasks/"]
```

Comment:
@kristoferbaxter could you take another look at my changes? i added a js dependency as well as having to turn the postHTML transformer asynchronous